server/controllers/notification.js

```diff
@@ -0,0 +1,180 @@
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
+
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  const {
+    user,
+    body: { notifyType, title, description, receivers },
```

Comment:
As a bonus, it would nice to validate the parameters from the request body and return a 400 if they're missing

Ditto througout!